undo https://github.com/torvalds/linux/commit/7188f656cdf762d4ea8ce16b6aaf4c6b06e119ec#diff-d66a8abf1bae6901f62be3c2621e673f
as it breaks jackd by setting a strange size fixed buffer size

it took me quite a while to find this as the cause of the problem ... :)

diff --git a/sound/soc/rockchip/rockchip_max98090.c b/sound/soc/rockchip/rockchip_max98090.c
index e80b09143b63..7df793748015 100644
--- a/sound/soc/rockchip/rockchip_max98090.c
+++ b/sound/soc/rockchip/rockchip_max98090.c
@@ -140,19 +140,19 @@ static int rk_aif1_hw_params(struct snd_pcm_substream *substream,
 	return ret;
 }
 
-static int rk_aif1_startup(struct snd_pcm_substream *substream)
-{
-	/*
-	 * Set period size to 240 because pl330 has issue
-	 * dealing with larger period in stress testing.
-	 */
-	return snd_pcm_hw_constraint_minmax(substream->runtime,
-			SNDRV_PCM_HW_PARAM_PERIOD_SIZE, 240, 240);
-}
+// static int rk_aif1_startup(struct snd_pcm_substream *substream)
+// {
+// 	/*
+// 	 * Set period size to 240 because pl330 has issue
+// 	 * dealing with larger period in stress testing.
+// 	 */
+// 	return snd_pcm_hw_constraint_minmax(substream->runtime,
+// 			SNDRV_PCM_HW_PARAM_PERIOD_SIZE, 240, 240);
+// }
 
 static const struct snd_soc_ops rk_aif1_ops = {
 	.hw_params = rk_aif1_hw_params,
-	.startup = rk_aif1_startup,
+// 	.startup = rk_aif1_startup,
 };
 
 SND_SOC_DAILINK_DEFS(hifi,
